specification/purview/data-plane/Azure.Analytics.Purview.PDS/preview/2022-11-01-preview/pds.json

```diff
@@ -0,0 +1,463 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Purview Distribution Service REST API Document",
+    "description": "Purview distribution service enables various Azure RPs to fetch the policies and attributes for data governance.",
+    "version": "2022-11-01-preview"
+  },
+  "paths": {
+    "/datasource-preferences": {
+      "get": {
+        "tags": [
+          "DatasourcePreferences"
+        ],
+        "description": "Get Datasource Preferences for a given scope.",
+        "operationId": "DatasourcePreferences_Get",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "query",
+            "name": "scope",
+            "description": "Scope.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "in": "query",
+            "name": "isProjectGondor",
+            "description": "Is call from a Gondor client.",
+            "type": "boolean"
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/DatasourcePreferencesResponse"
+            }
+          },
+          "default": {
+            "description": "An unexpected error occurred",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponseModel"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "DatasourcePreferences_Get": {
+            "$ref": "./examples/DatasourcePreferences_Get.json"
+          }
+        }
+      }
+    },
+    "/policy-elements": {
+      "get": {
+        "tags": [
+          "PolicyElements"
+        ],
+        "description": "Get policies for given scope.",
+        "operationId": "PolicyElements_Get",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "query",
+            "name": "scope",
+            "description": "Scope.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "in": "query",
+            "name": "api-version",
+            "description": "The API Version.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "in": "query",
+            "name": "filter",
+            "description": "filter.",
```

Comment:
Please describe the syntax/structure for specifying a filter. Do you support "full OData" syntax? A subset? Something else? Users need to know.

Reply:
Updated the description.